apps/judicial-system/web/src/components/Table/Table.tsx

```diff
@@ -180,6 +180,8 @@ const Table: FC<TableProps> = (props) => {
     switch (column) {
       case 'defendants':
         return entry.defendants?.[0]?.name ?? ''
+      case 'defendantsPunishmentType':
+        return entry.defendants?.[0]?.punishmentType ?? ''
```

Comment:
I would normally like to do this in a transformer in the api layer, but this is temporary code so it does not really matter in this case.

Reply:
As discussed async, we will remove this either when we:
- Represents multiple defendants per case differently in the client components
- Improve the table component, especially the sorting function, to make it more generic so we don't have to have these specific conditions in the table component itself.